j/js-cookie/js-cookie_ubi_8.5.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package	: js-cookie
+# Version	: v3.0.0
+# Source repo	: https://github.com/js-cookie/js-cookie
+# Tested on	: UBI: 8.5
+# Language      : Node
+# Travis-Check  : True
+# Script License: MIT License
```

Comment:
this is not the package license, every script in this repo is published under the Apache 2.0 license and hence this should always be Apache 2.0 for every script